src/commands/package-and-push.yaml

```diff
@@ -15,9 +15,13 @@ steps:
   - run: |
       git clone -q --depth=1 --single-branch -b master "git@github.com:giantswarm/$(cat .app_catalog_name).git" .app_catalog
   - run: |
-      helm package --save=false ./helm/<<parameters.chart>> --destination .app_catalog
+      helm init --client-only
   - run: |
-      helm repo index --url https://giantswarm.github.com/$(cat .app_catalog_name) .app_catalog
+      mkdir build && helm package ./helm/<<parameters.chart>> --destination ./build
+  - run: |
+      helm repo index --url https://giantswarm.github.com/$(cat .app_catalog_name) --merge .app_catalog/index.yaml ./build
+  - run: |
+      cp ./build/* .app_catalog
```

Comment:
Will this override index.yaml w/o any additional prompt?

Reply:
By this step ./build directory will contain merged index.yaml and chart tgz package.
Yes, copy will overwrite any existing index.yaml in .app_catalog directory. Next step commits that change of index.yaml and addition of the new package.
Both merge and copy + commit will work also on empty initialized catalog repo, even without index.yaml being there.